src/test/data/JsonSerializableAddressBookTest/typicalClientsAddressBook.json

```diff
@@ -1,68 +1,116 @@
 {
   "_comment": "AddressBook save file which contains the same Client values as in TypicalClients#getTypicalAddressBook()",
-  "clients" : [ {
-    "name" : "Alice Pauline",
-    "phone" : "94351253",
-    "email" : "alice@example.com",
-    "address" : "123, Jurong West Ave 6, #08-111",
-    "country" : "SG",
-    "timezone": "GMT+8",
-    "contractExpiryDate": "1-4-2021",
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Benson Meier",
-    "phone" : "98765432",
-    "email" : "johnd@example.com",
-    "address" : "311, Clementi Ave 2, #02-25",
-    "country" : "SG",
-    "timezone": "GMT+8",
-    "contractExpiryDate": "3-12-2022",
-    "tagged" : [ "owesMoney", "friends" ]
-  }, {
-    "name" : "Carl Kurz",
-    "phone" : "95352563",
-    "email" : "heinz@example.com",
-    "address" : "wall street",
-    "country" : "US",
-    "timezone": "GMT-4",
-    "contractExpiryDate": "30-1-2022",
-    "tagged" : [ ]
-  }, {
-    "name" : "Daniel Meier",
-    "phone" : "87652533",
-    "email" : "cornelia@example.com",
-    "address" : "10th street",
-    "country" : "SG",
-    "timezone": "GMT+8",
-    "contractExpiryDate": "28-2-2021",
-    "tagged" : [ "friends" ]
-  }, {
-    "name" : "Elle Meyer",
-    "phone" : "9482224",
-    "email" : "werner@example.com",
-    "address" : "michegan ave",
-    "country" : "GB",
-    "timezone": "GMT+1",
-    "contractExpiryDate": "10-10-2024",
-    "tagged" : [ ]
-  }, {
-    "name" : "Fiona Kunz",
-    "phone" : "9482427",
-    "email" : "lydia@example.com",
-    "address" : "little tokyo",
-    "country" : "JP",
-    "timezone": "GMT+9",
-    "contractExpiryDate": "9-11-2022",
-    "tagged" : [ ]
-  }, {
-    "name" : "George Best",
-    "phone" : "9482442",
-    "email" : "anna@example.com",
-    "address" : "4th street",
-    "country" : "US",
-    "timezone": "GMT-4",
-    "contractExpiryDate": "2-8-2021",
-    "tagged" : [ ]
-  } ],
-  "notes": []
+  "clients": [
```

Comment:
the formatting is still abit off, it should be `[ {` instead of `[<br>{` and `}, {` instead of `},<br>{`